src/main/resources/edu/cornell/kfs/tax/batch/DefaultTransactionRowOutputDefinition.xml

```diff
@@ -37,23 +37,27 @@
         <field name="Header25" length="14" type="STATIC" value="Foreign_Source" />
         <field name="Header26" length="19" type="STATIC" value="Fed_Inc_Tax_Percent" />
         <field name="Header27" length="100" type="STATIC" value="Payment_Description" />
-        <field name="Header28" length="45" type="STATIC" value="Payment_Address_Line_1" />
-        <field name="Header29" length="30" type="STATIC" value="Payment_Country_Name" />
-        <field name="Header30" length="10" type="STATIC" value="Chart_Code" />
-        <field name="Header31" length="14" type="STATIC" value="Account_Number" />
-        <field name="Header32" length="100" type="STATIC" value="Initiator_NetId" />
-        <field name="Header33" length="8" type="STATIC" value="1099_Box" />
-        <field name="Header34" length="19" type="STATIC" value="1099_Overridden_Box" />
-        <field name="Header35" length="10" type="STATIC" value="1042S_Box" />
-        <field name="Header36" length="20" type="STATIC" value="1042S_Overridden_Box" />
-        <field name="Header37" length="14" type="STATIC" value="Payment_Reason" />
+        <field name="Header28" length="55" type="STATIC" value="Payment_Address_Line_1" />
+        <field name="Header29" length="55" type="STATIC" value="Payment_Address_Line_2" />
+        <field name="Header30" length="50" type="STATIC" value="Payment_City" />
+        <field name="Header31" length="30" type="STATIC" value="Paymwnt_State" />

```

Comment:
Minor typo in "value" attribute; should not affect processing in this case, but it would be a good idea to correct this to avoid confusion for the tax managers.